packages/embarkjs/.gitignore

```diff
@@ -0,0 +1 @@
+dist
```

Comment:
shall we all package have dist folder in gitignore?
So we can make it at the root: Convention over Configuration

Reply:
We can.  In the beginnings of the monorepo, `packages/embark` has `dist/` and `packages/embark-ui` has `build/`. Common ignore-patterns were put together in the root `.gitignore` and patterns particular to a package were put in the package's `.gitignore`.  I didn't do a perfect job of it, but that was the theory.

If most of our packages will use `dist/` (makes sense) then we can add that pattern to the root and delete the file, since that's the only entry.

Reply:
Done in #1301.